pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -113,8 +113,8 @@ func (pl *CSILimits) Filter(ctx context.Context, _ *framework.CycleState, pod *v
 	// If CSINode doesn't exist, the predicate may read the limits from Node object
 	csiNode, err := pl.csiNodeLister.Get(node.Name)
 	if err != nil {
-		// TODO: return the error once CSINode is created by default (2 releases)
-		klog.V(5).InfoS("Could not get a CSINode object for the node", "node", klog.KObj(node), "err", err)
+		klog.V(5).ErrorS(err, "Could not get a CSINode object for the node", "node", klog.KObj(node))
+		return framework.AsStatus(err)
```

Comment:
So I think the original comment is outdated. This logic is also used by the cluster-autoscaler, which works by simulating Node objects and running scheduling plugins against the fake Node object. However, autoscaler does not support simulating CSINode objects today. So in order to make csi drivers mostly work today (which is not perfect), in the case of a CSINode object node existing we need to return success in order to make autoscaling succeed.